docs/scope.md

```diff
@@ -1,10 +1,151 @@
+| | +-----------------------+ | |            | | +-----------------------+ | |
+| |                           | |            | |                           | |
+| +---------------------------+ |            | +---------------------------+ |
+|                               |            |                               |
++-------------------------------+            +-------------------------------+
+```
+
+If you run Scope on the same machine as the Weave Network, the Probe will use
+WeaveDNS to automatically discover other Apps on your network. Scope acheives
+this by registering itself under the address scope.weave.local; each Probe
+will send reports to every App register for this address. Therefore, if
+you have a running WeaveDNS setup, you do not need to take any further steps.
+
+If you do not wish to use WeaveDNS, you can instruct Scope to cluster with
+other Scope instances on the command line. Hostnames and IP addresses are
+acceptable, both with and without ports:
 
 ```
 # weave launch scope1:4030 192.168.0.12 192.168.0.11:4030
 ```
+
+## Using Scope Service
+
+Scope can also be used to feed reports to the Scope Service. The Scope
+Service allows you centrally manage and share access to your Scope UI.  In this
+configuration, you only run the Probe locally - the Apps are hosted for you.
+
+To get an account on the Scope Service, sign up at scope.weave.works. You need
+to run a Probe on every machine you want to monitor with Scope. To launch a
+Probe and send reports to the service, run the following command:
+
+```
+sudo scope launch --service-token=<service token>
+```
+
+```
+                                 ___    ,'""""'.
+                             ,"""   """"'      `.
+                            ,'                   `._
+                           ,'                       `"""'.
+                          ,'                              `.
+                         ,'                                :
+                       ,'                                  `.
+                 ,""""'                                     `"""".
+               ,'                                                `.
+              ,'               scope.weave.works                  `.
+         ,-""'                                                    `.
+        ,'                                                          `.
+        ;            ,'                ','                            `.
+       ,'            ;                 ,'                              `.
+      ,'             ;                 (                    ,           `.
+      ;              `                `.                  ,'           ,'
+      ;               \                 `.                ,'    ,'    ,'
+      '.               '                 `.             ,''.   ,' `--'
+       `"-----"" _.-'' .-'`-.:..___...--' `-._      ,-"'   `-'
+                                      /\      `"""""
```

Comment:
@squaremo I was told you might have some cloud ascii art which looks less like a brain with testicles?